### PR TITLE
Added apache pulsar to projects using etcd.

### PR DESCRIPTION
**Pull request by liangyuanpeng**
Apache pulsar support etcd as a metadata store, here is the doc.
https://pulsar.apache.org/docs/3.0.x/administration-metadata-store/#use-etcd-as-metadata-store

### CODE REVIEW

**Review by jmhbnz — COMMENTED** [review]
Hey @liangyuanpeng - Thanks for letting us know about this :)

Please ensure your commit is signed so the DCO check passes.

```bash
git rebase HEAD~1 --signoff
git push --force
```

**Review by jmhbnz — APPROVED** [review]
LGTM

File: content/en/docs/v3.5/integrations.md [hunk 1]
```diff
@@ -197,3 +197,4 @@ The sections below list etcd client libraries by language.
 - [Apache APISIX](https://github.com/apache/apisix) - Apache APISIX is a dynamic, real-time, high-performance API gateway.
 - [purpleidea/mgmt](https://github.com/purpleidea/mgmt) - Next generation distributed, event-driven, parallel config management!
 - [Portworx/kvdb](https://docs.portworx.com/concepts/internal-kvdb/) - The internal kvdb for storing Portworx cluster configuration.
+- [Apache Pulsar](https://apache/pulsar) - Apache Pulsar is an open-source, distributed messaging and streaming platform built for the cloud.
```
Comment (ahrtr) [inline content/en/docs/v3.5/integrations.md:200]: Are you sure "https://apache/pulsar" is the correct link?

**Review by spzala — COMMENTED** [review]
Thanks @jmhbnz The `https://pulsar.apache.org/` looks good to me. 

@ahrtr good eyes :)

File: content/en/docs/v3.5/integrations.md (hunk 1 above)
Comment (liangyuanpeng) [reply content/en/docs/v3.5/integrations.md]: Yes, This is a mistake, thanks.

**Review by ahrtr — APPROVED** [review]

**Comment by liangyuanpeng** [comment]
@spzala PTAL,Thanks.

**Pull request merged**